docs/versioned_docs/version-1.0/30-administration/22-backends/40-kubernetes.md

```diff
@@ -40,6 +40,12 @@ Additional labels to apply to worker pods. Must be a YAML object, e.g. `{"exampl
 
 Additional annotations to apply to worker pods. Must be a YAML object, e.g. `{"example.com/test-annotation":"test-value"}`.
 
+### `WOODPECKER_BACKEND_K8S_PULL_SECRET_NAMES`
+> Default: `regcred`
```

Comment:
Not necessarily, but removing it would cause a breaking change for users already using it unless they explicitly define `WOODPECKER_BACKEND_K8S_PULL_SECRET_NAMES=regcred`. 

One option would be to deprecate it for a point release and remove the default for the next major version.

Reply:
It's done already.

https://github.com/woodpecker-ci/woodpecker/blob/507a6f1753c9feba3aa572283dbd14dd340bf95c/pipeline/backend/kubernetes/flags.go#L81

Reply:
Now I get it 😆 Will do.